modules/new_serverpod_auth/serverpod_auth_core/serverpod_auth_core_client/lib/src/protocol/protocol.dart

```diff
@@ -182,8 +182,8 @@ class Protocol extends _i1.SerializationManager {
         return 'TokenPair';
       case _i13.UserProfileModel():
         return 'UserProfileModel';
-      case _i14.AuthSessionInfo():
-        return 'AuthSessionInfo';
+      case _i14.ServerSideSessionInfo():
```

Comment:
Question: Is this the dual for the `JwtTokenInfo`? If so, shouldn't it be called `<sssPrefix>TokenInfo` as well?

```suggestion
      case _i14.ServerSideSessionTokenInfo():
```
> This suggestion is only here to highlight the proposal, not to be applied.